docs/mapping.md

```diff
@@ -0,0 +1,292 @@
+(mapping)=
+# Mapping configuration
+
+## Introduction
+
+A mapping configuration is made of 3 objects: 
+- an IIDM network
+- a time series store (in csv format)
+- a mapping groovy script using the [mapping DSL](#mapping-dsl) described below
+
+The mapping process is mainly used through Metrix simulator, but it may be executed separately in order to check its intermediate results.
+
+With only required options, the mapping does not produce any output. In adding option `--mapping-synthesis-dir`, it will output a [synthesis](#synthesis) of the mapping, allowing you to check the consistency of the produced multi cases. If you want to go deeper in the mapping process, the option `--check-equipment-time-series` will produce the full time series mapping. Lastly, the option `--network-output-dir` will output the network case with mapped values for the steps requested.
+
+## Mapping DSL
+
+The purpose of the mapping is to map time series values on network elements. As it is a groovy script, it can create composite time series or modify the network.
+
+### General parameters
+
+The general parameters are used to configure globally the mapping.
+
+```groovy
+parameters {
+    toleranceThreshold // Tolerance threshold when comparing the mapped power to the limit defined by the network element (default : 0.0001) 
+}
+``` 
+
+### Time series mapping to network
+
+There is one mapping function for each type of network elements:
+
+```groovy
+mapToGenerators {...}
+mapToLoads {...}
+mapToHvdcLines {...}
+mapToBoundaryLines {...}
+mapToBreakers {...}
+mapToTransformers {...}
+mapToPhaseTapChangers {...}
+mapToRatioTapChangers {...}
+mapToLccConverterStations {...}
+mapToVscConverterStations {...}
+```
+
+the general syntax is: 
+
+```groovy
+mapToXXX {
+ variable variableName //optional, the name of the element variable to map the time series with. Each element have a default mapped variable (defined below)
+ timeSeriesName 'timeseriesName' // name of the time series to map
+ filter {...} // a filter predicate allowing to select which item of the element type we wish to map
+ distributionKey {...} // optional, a repartition key that will define how the time serie values will be distributed for each selected items (default is equipartition : val / N)
+}   
+```
+The element of the general syntax are described below. 
+
+Note that for the `mapToBreakers` function, the `variable` and `distributionKey` are always ignored.
+
+#### Time series
+
+With the `timeSeriesName` variable you can map any time series by referring to its name. The available time series 
+must exist in the input data set or be created within the groovy script. For more details about time series management, 
+refer to the [time series description](inv:core:std:doc#data/timeseries).
```

Comment:
With this configuration of the cross-project links (intersphinx), the text of the link is not kept when the index of the external site is not found (error [myst.iref_missing]).

In this case, it will render as:

> ![image](https://github.com/powsybl/powsybl-metrix/assets/44769592/1f325319-6532-425e-a99b-ce8dcc717b8c)

I don't know if it is configurable or if we should use something like `refer to the time series description in the powsybl-core documentation[ (See here)](inv:core:std:doc#data/timeseries).` instead.
This is ugly, but it won't remove useful text if the link is not found.

Reply:
I think it will mainly be an issue for the first versions of the documentation. Once the first versions are published, it's for the writers to check that their links works.

Reply:
Ok, we will see later if a better solution is necessary, depending if this is really a problem after the 1st versions are published.